frankenstein/prog/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/prometheus/common/log"
+	"github.com/prometheus/prometheus/frankenstein"
+)
+
+func main() {
+	var (
+		listen       string
+		consulHost   string
+		consulPrefix string
+	)
+
+	flag.StringVar(&listen, "listen", ":80", "HTTP server listen address")
```

Comment:
Flag descriptions in Prometheus should end with a period.

The usual flag name for the HTTP server address is `-web.listen-address` (hey, there might even be a debug web interface on it!).